src/applications/accredited-representative-portal/routes.jsx

```diff
@@ -22,10 +25,32 @@ const router = createBrowserRouter(
             {
               path: 'poa-requests',
               element: <POARequestsPage />,
+              loader: poaRequestsLoader,
+              errorElement: <ErrorMessage />,
+              children: [
+                {
+                  index: true,
+                  loader: poaRequestsLoader,
+                  element: <POARequestsCard cssClass="pending" />,
+                },
+                {
+                  index: true,
+                  path: 'pending',
+                  loader: poaRequestsLoader,
+                  element: <POARequestsCard cssClass="pending" />,
+                },
+                {
+                  path: 'completed',
+                  // loader: poaRequestsCompletedLoader,
+                  element: <POARequestsCard cssClass="completed" />,
+                },
+              ],
```

Comment:
- We still want the previous route structure
- Tab queries go in query parameters
- The Card component is for a single POA request, `POARequestsPage` is for rendering a list of cards

Reply:
updated